<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddRoomWnd</class>
 <widget class="QWidget" name="AddRoomWnd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>122</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddRoomWnd</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>房间类型</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="typeBox">
         <item>
          <property name="text">
           <string>标间</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>单间</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>套房</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>房间号</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IDEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="noteLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>IDEdit</zorder>
  <zorder>typeBox</zorder>
  <zorder>label_4</zorder>
  <zorder>label_7</zorder>
  <zorder>addBtn</zorder>
  <zorder>addBtn</zorder>
  <zorder>noteLabel</zorder>
  <zorder>addBtn</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
